translations/handleExternalBranchedAndroidTranslations.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# verbose and exit on error
+set -xe
+
+# import GPG keys
+gpg --import /gpg/nextcloud-bot.public.asc
+gpg --allow-secret-key-import --import /gpg/nextcloud-bot.asc
+gpg --list-keys
+
+# fetch git repo
+git clone git@github.com:$1/$2 /app
+
+# switch to translation branch
+git checkout $3
```

Comment:
In my approach, I assumed that the branch already exists and is not deleted during merge, since deleting has to be done manually. However, I changed the script to be applicable for both cases.